data_portal/fields.py

```diff
@@ -45,3 +47,10 @@ def add(self, value):
 
     def calculate_hash(self):
         return self.__sha256.hexdigest()
+
+
+class IdHelper(object):
```

Comment:
With the introduction of this `portal_run_id` generator function, should the `WorkflowHelper` not use this as well?
(now we have two places in which the IDs can be generated)

Reply:
Yup. That's need follow up there with changes in domain package. Plan is the `WorkflowHelper` needs to refactor to use this one.